### PR TITLE
design landing page

### PR DESCRIPTION
**Pull request by magicmarie**
### What does this PR do?
Implements landing page for Events manager

### Description of Task to be completed?
- Edit the landing page to contain buttons for the login and signup
- Style the landing page according to the mockups

### CODE REVIEW

File: src/components/home.js [hunk 1]
```diff
@@ -1,12 +1,50 @@
...
+    this.props.getEventsAction();
+  }
 
-export default Home;
+  render() {
+    const events = this.props.events;
+    const eventsList = events.length ? (
+      events.map(event => {
+        return (
+          <div className="event" key={event.id}>
```
Comment (jokamjohn) [inline src/components/home.js]: @magicmarie can you move all the content in the map into a component

File: src/containers/index.js [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
- class App extends Component{
-    render() {
-        return(
-                <Routes/>
-        );
-    }
+class App extends Component {
+  render() {
+    return (
+      <div>
```
Comment (jokamjohn) [inline src/containers/index.js]: @magicmarie why did you include all these divs, you should only render <Routes /> here

File: src/index.html [hunk 3]
```diff
@@ -6,9 +6,39 @@
...
+        crossorigin="anonymous">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     </head>
     <body>
         <div id="App">
         </div>
+
+        <script>
+            // Your web app's Firebase configuration
+            var firebaseConfig = {
```
Comment (jokamjohn) [inline src/index.html]: @magicmarie these configurations should not be in this file. They should be in a .js file in your src directory. And also use env variables for the config values.

File: src/index.js [hunk 4]
```diff
@@ -4,8 +4,17 @@ import { Provider } from "react-redux";
 import store from "../src/js/store/index";
 import App from "./containers/index";
 
+// Firebase App (the core Firebase SDK) is always required and
+// must be listed before other Firebase SDKs
+var firebase = require("firebase/app");
```
Comment (jokamjohn) [inline src/index.js]: @magicmarie These firebase lines should be in another firebase js file with the Firebase configs

File: src/js/actions/getEvents.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+import { FETCH_EVENTS_SUCCESS } from "./types";
+
+export const getEventsAction = () => dispatch => {
+  return axios
```
Comment (jokamjohn) [inline src/js/actions/getEvents.js]: @magicmarie You should create an axios instance, then call the different http verb methods from it

File: src/index.html (hunk 3 above)
Comment (magicmarie) [reply src/index.html]: oh yes.. intended to do that..

**Comment by magicmarie** [comment]
@jokamjohn feedback implemented.

File: src/Firebase.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    projectId: process.env.projectId,
+    storageBucket: process.env.storageBucket,
+    messagingSenderId: process.env.messagingSenderId,
+    appId: process.env.appId
+  };
+
+  // Initialize Firebase
+  class Firebase{
+    constructor() {
+        app.initializeApp(firebaseConfig);
```
Comment (jokamjohn) [inline src/Firebase.js]: @magicmarie you should check whether there is no other Firebase app instance before initializing another app.
Consider the code below
```
if (!firebase.apps.length) {
  firebase.initializeApp(config);
}
```

**Pull request merged**